modules/introduction/partials/new-features-80.adoc

```diff
@@ -59,6 +59,23 @@ curl --get -u <username:password> \
     -d clusterLabels=none|uuidOnly|uuidAndName
 ----
 
+[#section-new-feature-800-services]
+=== Services
+
+https://jira.issues.couchbase.com/browse/MB-60349[MB-60349]::
+Now you can add or remove non-Data Services dynamically on existing nodes in a cluster, without adding or removing nodes.
+Rebalancing is automatically triggered to evenly distribute service workloads.
```

Comment:
I don't think we should say "evenly distribute" -- how each server service handles the distribution of workloads is different from service to service -- as described here -- 
https://docs.couchbase.com/server/current/learn/clusters-and-availability/rebalance.html#rebalance-and-other-services

Might be better to say:
Rebalancing is automatically triggered to distribute service workloads and to complete the change.

cc @anuthan